Controller/CRUDController.php

```diff
@@ -99,7 +99,7 @@ public function listAction()
             $this->admin->setListMode($listMode);
         }
 
-        $datagrid = $this->admin->getDatagrid();
+        $datagrid = $this->get('sonata.admin.admin_builder')->getDatagrid($this->admin);
```

Comment:
It could not be BC if someone was extending admin and redefining getDatagrid.

I am not sure how should we handle it then, because if we don't change this, we cannot deprecate getDatagrid.